Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityDeviceManager.cs

```diff
@@ -393,6 +393,8 @@ public override void Enable()
             {
                 GestureRecognizerEnabled = true;
             }
+
+            IsEnabled = true;
```

Comment:
Not entirely sure why we don't call base.Enable() here vs. setting the bool.

Reply:
There's a few other places in this commit where this occurs so just want to ensure consistency between manually setting vs. relying on the base class function.

Reply:
essentially the same reason that isinitialized is being set directly. 100% happy to switch this to a call to the base class

Reply:
updated to use base.Enable and added comments when it was "out of place"